scripts/workers/rl_submission_worker.py

```diff
@@ -117,6 +117,8 @@ def main():
                 if submission.get("status") == "finished":
                     message_receipt_handle = message.get("receipt_handle")
                     api.delete_message_from_sqs_queue(message_receipt_handle)
+                    if submission.challenge_phase.challenge.is_docker_based:
+                        pass
```

Comment:
`submission.challenge_phase.challenge.is_docker_based` condition not required here.